src/components/gallery/GalleryItem.tsx

```
@@ -0,0 +1,50 @@
+
+interface GalleryItemProps {
+  template: Template;
+  onClick: (template: Template) => void;
+}
+
+function GalleryItem({ template, onClick }: GalleryItemProps) {
+  return (
+    <div
+      className="group relative cursor-pointer overflow-hidden"
+      onClick={() => onClick(template)}
+    >
```

Biome

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)